packages/rspack-test-tools/tests/statsAPICases/to-string.js

```diff
@@ -11,27 +11,13 @@ module.exports = {
 		expect(stats?.toString({ timings: false, version: false }))
 			.toMatchInlineSnapshot(`
 		"PublicPath: auto
-		asset main.js 692 bytes [emitted] (name: main)
-		Entrypoint main 692 bytes = main.js
+		asset main.js 353 bytes [emitted] (name: main)
+		Entrypoint main 353 bytes = main.js
 		./fixtures/a.js
 		./fixtures/b.js
 		./fixtures/c.js
 		./fixtures/abc.js
-
-		ERROR in ./fixtures/b.js
-		  × Module parse failed:
-		  ╰─▶   × JavaScript parsing error: Return statement is not allowed here
-		         ╭─[4:1]
-		       4 │
-		       5 │ // Test CJS top-level return
-		       6 │ return;
-		         · ───────
-		         ╰────
-		      
-		  help: 
-		        You may need an appropriate loader to handle this file type.
-
-		Rspack compiled with 1 error (9f555e6a76ee5e3780ec)"
```

Comment:
This diagnostic is not correct. It's `"javascript/auto"`, which can contain top-level return if it is regarded as a `"script"`